flaml/tune/tune.py

```diff
@@ -564,6 +567,12 @@ def easy_objective(config):
             searcher.set_search_properties(metric, mode, config, **setting)
         else:
             searcher.set_search_properties(metric, mode, config)
+        if lexico_objectives:
+            assert search_alg.__class__.__name__ in [
+                "BlendSearch",
+                "CFO",
+            ], "If lexico_objectives is not None, search_alg must be CFO or BlendSearch."
+            search_alg.lexico_objectives = lexico_objectives

```

Comment:
Shall we implement this in `set_search_properties`? Otherwise we need to document this way of setting `lexico_objectives` somewhere. Conceptually I think it should be part of `set_search_properties`.

Reply:
Yes. I agree. I support it in CFO for now. Will make it work in BlendSearch in future PR.